### PR TITLE
plasmid workflow

### PR DESCRIPTION
**Pull request by dev-rajneeshkumar**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--moleculardevices--hlxsites.hlx.page/applications/plasmid#workflow
- After: https://plasmid-workflow--moleculardevices--hlxsites.hlx.page/applications/plasmid#workflow

### CODE REVIEW

File: fragments/applications/automated-workflow-for-plasmid-production.html [hunk 1]
```diff
@@ -0,0 +1,169 @@
+<link rel="stylesheet" href="/fragments/applications/application-workflow-commons.css" />
+<style>
+  #conference-timeline .conference-center-line {
+    height: 74%;
+  }
```
Comment (mhaack) [inline fragments/applications/automated-workflow-for-plasmid-production.html:5]: Why is this here? Can this go into the .css file?

Comment (dev-rajneeshkumar) [reply fragments/applications/automated-workflow-for-plasmid-production.html]: changes moved to css @mhaack
